indico/web/templates/footer.html

```diff
@@ -40,6 +42,7 @@
                 {% endblock %}
             </div>
         </div>
+        {{ setup_session_expiration() }}
```

Comment:
Wouldn't `indico_base.html` be better to be consistent with it also being added in `indico_new_base.html`?

Reply:
Yeah that's much better. I think since `<!DOCTYPE html>` should always come at the top, I've put the import on the second line.

Reply:
jinja imports do not show up in the final code. And when using `{%- ...  -%}` any surrounding whitespace will be swallowed

Reply:
okay